src/web/CarplayWeb.ts

```diff
@@ -89,6 +89,11 @@ export default class CarplayWeb {
   start = async (usbDevice: USBDevice) => {
     if (this._started) return
     const { initialise, open, send } = this.dongleDriver
+
+    console.debug('opening device')
```

Comment:
Moved open/reset outside of the dongle driver, so we can handle it differently in Web vs Node. In Web, we just open and reset it so `initialize` gets an already-opened device, in Node, we open then reset it, then close and wait for it to disappear and reappear and then open it again to give to `initialize`